packages/create-workleap/prompts.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  isCancel,
+  cancel,
+  intro,
+  outro,
+  text,
+  confirm,
+  select,
+  multiselect,
+  note,
+} from "@clack/prompts";
+import pc from "picocolors";
+
+const DEFAULT_CANCEL_MESSAGE = "Operation cancelled.";
+
+type Primitive = Readonly<string | boolean | number>;
+export type Option<Value> = Value extends Primitive
+  ? {
+      value: Value;
+      label?: string;
+      hint?: string;
+    }
+  : {
+      value: Value;
+      label: string;
+      hint?: string;
+    };
+
+export const Intro = (title?: string) => {
```

Comment:
Class names should be PascalCase
Types should be PascalCase
Function/Arrow functions should be camelCase